### PR TITLE
Added ext-xml to require in composer.json file

### PR DESCRIPTION
**Pull request by WhiteEagle88**

### CODE REVIEW

**Comment by ravage84** [comment]
Which part of the PHPMD uses this?

The JSON does not look valid, by the way...

**Comment by WhiteEagle88** [comment]
@ravage84 
"Which part of the PHPMD uses this?":
- https://github.com/phpmd/phpmd/blob/c51ea72761fdaf38c1787658f07e2d20b1141ae0/src/site/scripts/update-rules.php#L41
- https://github.com/phpmd/phpmd/blob/c51ea72761fdaf38c1787658f07e2d20b1141ae0/src/main/php/PHPMD/RuleSetFactory.php#L231
- https://github.com/phpmd/phpmd/blob/c51ea72761fdaf38c1787658f07e2d20b1141ae0/src/main/php/PHPMD/RuleSetFactory.php#L551
- https://github.com/phpmd/phpmd/blob/c51ea72761fdaf38c1787658f07e2d20b1141ae0/src/test/php/PHPMD/AbstractTest.php#L473

"The JSON does not look valid, by the way...": What do you mean?

File: composer.json [hunk 1]
```diff
@@ -36,7 +36,8 @@
     "minimum-stability" : "stable",
     "require": {
         "php": ">=5.3.9",
-        "pdepend/pdepend": "^2.2"
+        "pdepend/pdepend": "^2.2",
+        "ext-xml": *
```
Comment (ravage84) [inline composer.json]: https://getcomposer.org/doc/04-schema.md#package-links

````
"ext-xml": "*"
````

With double quotes.

**Comment by ravage84** [comment]
Thanks for pointing that out. I'll merge it, once the JSON is fixed.

**Comment by WhiteEagle88** [comment]
Oh, sorry. I fixed, thanks.

**Comment by ravage84** [comment]
Thanks!

**Pull request merged**